### PR TITLE
Makes plushies huggable to reduce stress from monophobia

### PR DESCRIPTION
**Pull request by NithaIsTired**
## About The Pull Request

Monophobia will now check for your hands for your pets/plushies too.
Rebalances the cost to be more in pair with RDS, given it not only has hallucinations symptoms but can also kill you instantly if you are unlucky.
Makes you able to hug plushies for companion, trading off having to have a full hand for it.

## Why It's Good For The Game

Adds a new way to play with the monophobia quirk, making it more appealing to be used. Also adds another use for plushies, as they don't usually do much other than make noise.

You will be able to hold plushies for comfort if you have monophobia, reducing your stress. This makes sure you can play the game without randomly dying to a heart attack and encourages the quirk to be roleplayed in a unique way too.

It's more fun than having to rely on a random cargo pet that you took a while to get just to not really care about it and only use it to not feel alone. Plushies with unique names are usually not replaceable, so that's more fun having to care about something you can put at least a bit of lore into!

The plushies are a cheaper alternative other than carrying pets around. They are flammable, can easily be stolen and any sharp tool will rip out their stuffing, making them not comfortable to hug anymore. You can't just toss the plushie in a corner, you do need it close to you for any stress to be reduced.

Also gives you a nice excuse to be carrying a big shark plushie 24/7.
dreamseeker_j3rDCTCW12
dreamseeker_Y3S9PHfLHI

## Changelog

:cl:
add: You can now hug plushies to reduce stress if you have monophobia.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/game/brain_damage/severe.dm [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			return FALSE
+	for(var/obj/item/toy/plush/plush in  owner.held_items) //Now you have an excuse to carry a shark plushie around all the time
+		if(plush.stuffed)
+			return FALSE
+	for(var/mob/M in owner.held_items) //makes sure to check hands for your tiny friends!
+		if(!isliving(M))
+			continue
+		if(istype(M, /mob/living/simple_animal/pet) || istype(M, /mob/living/basic/pet) || M.ckey)
+			return FALSE
+	return TRUE
```
Comment (Majkl-J) [inline modular_zubbers/code/game/brain_damage/severe.dm:21]:
```suggestion
	. = ..()
	for(var/obj/item/toy/plush/plush in oview(owner, 1))
		if(plush.stuffed)
			return FALSE
	for(var/obj/item/toy/plush/plush in  owner.held_items) //Now you have an excuse to carry a shark plushie around all the time
		if(plush.stuffed)
			return FALSE
	for(var/mob/M in owner.held_items) //makes sure to check hands for your tiny friends!
		if(!isliving(M))
			continue
		if(istype(M, /mob/living/simple_animal/pet) || istype(M, /mob/living/basic/pet) || M.ckey)
			return FALSE
```

File: modular_zubbers/code/game/Items/plushes.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/obj/item/toy/plush/attack_self(mob/user)
+	. = ..()
+	if(stuffed || grenade)
+		to_chat(user, span_notice("You pet [src]. D'awww."))
+		if(grenade && !grenade.active)
+			user.log_message("activated a hidden grenade in [src].", LOG_VICTIM)
+			grenade.arm_grenade(user, msg = FALSE, volume = 10)
```
Comment (Majkl-J) [inline modular_zubbers/code/game/Items/plushes.dm:7]:
```suggestion
	if(stuffed)
```

File: modular_zubbers/code/game/Items/plushes.dm [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+		if(HAS_TRAIT(user, TRAIT_MONOPHOBIA))
+			to_chat(user, span_notice("You feel your heart warm up... You don't feel so alone."))
+
+	else
+		to_chat(user, span_notice("You try to pet [src], but it has no stuffing. Aww..."))
```
Comment (Majkl-J) [inline modular_zubbers/code/game/Items/plushes.dm:12]:
```suggestion
```

File: code/game/objects/items/plushes.dm [hunk 4]
```diff
@@ -112,7 +112,7 @@
 	. = ..()
 	if(gone == grenade)
 		grenade = null
-
+/* BUBBER EDIT - MOVED TO ZUBBERS
```
Comment (Majkl-J) [inline code/game/objects/items/plushes.dm:115]:
```suggestion
```

File: code/game/objects/items/plushes.dm [hunk 5]
```diff
@@ -121,7 +121,7 @@
 			user.log_message("activated a hidden grenade in [src].", LOG_VICTIM)
 			grenade.arm_grenade(user, msg = FALSE, volume = 10)
 	else
-		to_chat(user, span_notice("You try to pet [src], but it has no stuffing. Aww..."))
+		to_chat(user, span_notice("You try to pet [src], but it has no stuffing. Aww..."))*/
```
Comment (Majkl-J) [inline code/game/objects/items/plushes.dm:124]:
```suggestion
		to_chat(user, span_notice("You try to pet [src], but it has no stuffing. Aww..."))
```

File: code/datums/brain_damage/severe.dm [hunk 6]
```diff
@@ -189,7 +189,7 @@
 			continue
 		if(istype(M, /mob/living/simple_animal/pet) || istype(M, /mob/living/basic/pet) || M.ckey)
 			return FALSE
-	return TRUE
+	return TRUE*/
```
Comment (Majkl-J) [inline code/datums/brain_damage/severe.dm:192]:
```suggestion
	return TRUE
```

File: code/datums/brain_damage/severe.dm [hunk 7]
```diff
@@ -179,7 +179,7 @@
 			stress_reaction()
 	else
 		stress = max(stress - (2 * seconds_per_tick), 0)
-
+/* BUBBER EDIT - MOVED TO ZUBBERS
```
Comment (Majkl-J) [inline code/datums/brain_damage/severe.dm:182]:
```suggestion
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Ohk so I have no idea if I did this correctly but this should help you get it working

Worst case: https://www.byond.com/forum/post/2093473

**Comment by BurgerLUA** [comment]
god this makes me sad

**Comment by YakumoChen** [comment]
> It's more fun than having to rely on a random cargo pet that you took a while to get just to not really care about it and only use it to not feel alone. Plushies with unique names are usually not replaceable, so that's more fun having to care about something you can put at least a bit of lore into!

The plushies are a cheaper alternative other than carrying pets around. They are flammable, can easily be stolen and any sharp tool will rip out their stuffing, making them not comfortable to hug anymore. You can't just toss the plushie in a corner, you do need it close to you for any stress to be reduced.

I don't get how it's more "fun" to replace pets with plushies? Unless "fun" is just a convenient word for "lazy"

I haven't ever seen someone steal a plushie from another player (pet theft is much more common) and pets are a lot easier to ruin (kill) than the very specific and almost-never-happens methods to ruin plushies

"It's cheaper" isn't a valid reason either

File: modular_zubbers/code/game/brain_damage/severe.dm [hunk 8]
```diff
@@ -0,0 +1,13 @@
+/datum/brain_trauma/severe/monophobia/check_alone()
+	. = ..()
+	for(var/obj/item/toy/plush/plush in oview(owner, 1))
```
Comment (nevimer) [inline modular_zubbers/code/game/brain_damage/severe.dm:3]: The code may always stop here, as if this is met, I don't think the held item check can even be reached.

**Review by nevimer — APPROVED** [review]

File: modular_zubbers/code/game/brain_damage/severe.dm (hunk 8 above)
Comment (NithaIsTired) [reply modular_zubbers/code/game/brain_damage/severe.dm:3]: I tested it out, it does seem to reach the held item check. I know it didn't check hands from experience trying to hold pets and suddenly getting stressed again, that's why I ended up adding that check.

**Pull request merged**